lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1561,6 +1561,15 @@ def self.notify
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of the aws_sdk_lambda library at start-up. May be one of `auto`, `prepend`, `chain`, `disabled`.'
         },
+        :'instrumentation.kinesis' => {
```

Comment:
This is a great call out! Consistency is important. I like the idea of using the gem name. I think in most places that is what we do that. 

Although I dont think we use hypens in config names, so this would be `instrumentation.aws_sdk_kinesis`. What do you think? cc @tannalynn

Reply:
`instrumentation.aws_sdk_kinesis` looks good to me. Curious to hear @tannalynn's thoughts on the naming approach overall.

Reply:
The name has been udpated.

Reply:
oh i didnt see this till now, oops! but yeah that sounds good to me too. As for the inconsistency on the aws namings, that's my bad, sorry about that! I didn't think about that until now.